services/ctan/ctan.tester.js

```diff
@@ -8,6 +8,10 @@ const isVPlusDottedVersionAtLeastOneWithOptionalAlphabetLetter = withRegex(
   /^v\d+(\.\d+)?(\.\d+)?[a-z]?$/
 )
 
+// a loose pattern matches an ISO 8601 date
+// e.g.: 2023-01-01
+const isIsoDate = withRegex(/^\d{4}-\d{2}-\d{2}$/)
```

Comment:
I don't think we need to make a custom validator here. We should be able to use `Joi.date().iso()`

Reply:
Ah, I just don't know `Joi.date().iso()` can also be used here.